### ISSUE TITLE
normalization of inputs

### ISSUE THREAD

**Issue by nationalflag** [kenshohara/3D-ResNets-PyTorch#7]
Did you normalize the inputs to [0,1] in the training phases of the pretrained models?

**Comment by kenshohara** [comment]
No, I did not use such normalization.
I use the inputs as [0-255].
